include/boost/gil/extension/toolbox/metafunctions/pattern.hpp

```diff
@@ -0,0 +1,64 @@
+// Copyright Tom Brinkman 2008. Distributed under the Boost
+// Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef _pattern_hpp_
+#define _pattern_hpp_
```

Comment:
These header guards need to be changed, see https://github.com/boostorg/gil/blob/develop/CONTRIBUTING.md#guidelines